### DISCUSSION TITLE
Support for scrobbling to Last.fm

### DISCUSSION THREAD

**Discussion by 24hourbullshit** [music-assistant/support#1107] [Feature requests and ideas]
I noticed that Music Assistant in conjunction with LMS (Logitech Media Server) doesn't scrobble to Last.fm.

At least not for me.

So it would be nice with support for scrobbling to Last.fm.

UPDATE: Suddenly it (LMS) seems to scrobbling, but not very consistent, for some reason...don't know if it's a LMS issue or a MA issue not sending correct tags or whatever to LMS.

Either way it would be nice with native Last.fm support.

**Comment by marcelveldt** [comment]
Good idea, MA v2 has a plugin framework so relative easy to add this.

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/632
